CHANGELOG.md

```diff
@@ -4,6 +4,88 @@ All notable changes to this project will be documented in this file.  See
 [standard-version](https://github.com/conventional-changelog/standard-version)
 for commit guidelines.
 
+## Not yet released
+
+### Features and enhancements
+
+* [ide] Add a feature to install libraries right from IDE. Hit “File → Add
+  Library”, enter a name as seen at https://xod.io/libs/, and you’re ready to
+  use new nodes.
+* [ide] Add a feature to publish your project as a library. Hit “File →
+  Publish Library” and you’re done. See your library at https://xod.io/libs/.
+  Others can immediately install it for themselves.
+* [ide] Add account pane to login/logout from within IDE. Hit “View →
+  Toggle Account Pane” to access it. Login is required to publish a library.
+* [ide] Embed C++ editor for low-level nodes. Double-click a
+  `not-implemented-in-xod` node to open and edit the code.
+* [ide] Click and drag for bulk selection with a marquee. Move, copy, paste,
+  delete faster. Drag from left to right to select only items completely
+  covered by the rectangle, drag from right to left to also include intersected
+  entities. Hold Ctrl key to add/remove items from the selection.
```

Comment:
Ctrl or Cmd...
Or just assume that Mac users default read Ctrl as Cmd? :)

Reply:
Done.

I thought the brain of a macoser does that conversion automatically :grimacing: